client/src/app/shared/hooks/use-table-controls/useTableControlProps.ts

```diff
@@ -173,6 +173,25 @@ export const useTableControlProps = <
     },
   });
 
+  const getExpandTdProps = ({
+    item,
+    rowIndex,
+  }: {
+    item: TItem;
+    rowIndex: number;
+  }): Omit<TdProps, "ref"> => ({
+    expand: {
+      rowIndex,
+      isExpanded: isCellExpanded(item),
+      onToggle: () =>
+        setCellExpanded({
+          item,
+          isExpanding: !isCellExpanded(item),
+        }),
+      expandId: "expandable-example",
```

Comment:
These ids need to be unique for QE tests. We probably want to be able to pass in a custom id here, or maybe have it be something built-in like:
```ts
expandId: `expandable-row-${item.name}`
```